rm-datasource/src/main/java/io/seata/rm/datasource/sql/struct/cache/PostgresqlTableMetaCache.java

```diff
@@ -100,11 +100,23 @@ private TableMeta resultSetMetaToSchema(DatabaseMetaData dbmd, String tableName)
          * select * from "Select"
          * select * from "Sel""ect"
          * select * from "Sel'ect"
+         * select * from TEST.test
+         * select * from test.TEST
+         * select * from "Test".test
+         * select * from "Test"."Select"
          */
         if (null != schemaName) {
-            schemaName = schemaName.replaceAll("(^\")|(\"$)", "");
+            if (schemaName.startsWith("\"") && schemaName.endsWith("\"")) {
+                schemaName = schemaName.replaceAll("(^\")|(\"$)", "");
```

Comment:
Can quotes exist between characters?

Reply:
It has been confirmed that it may contain special characters.

Reply:
support select * from "Te"'st"."Select"